app/src/main/res/layout/layout_lecture_selection.xml

```diff
@@ -6,58 +6,55 @@
     android:layout_gravity="center"
     android:orientation="horizontal">
 
-    <LinearLayout
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:orientation="vertical">
-
-        <Button
-            android:id="@+id/lecture_1"
-            android:layout_width="@dimen/next_lecture_select_width"
-            android:layout_height="@dimen/next_lecture_select_width"
-            android:layout_margin="1dp"
-            android:background="@drawable/next_lecture_select"
-            android:text="1"
-            android:textColor="#ffffffff"
-            tools:ignore="HardcodedText" />
-
-        <Button
-            android:id="@+id/lecture_2"
-            android:layout_width="@dimen/next_lecture_select_width"
-            android:layout_height="@dimen/next_lecture_select_width"
-            android:layout_margin="1dp"
-            android:background="@drawable/next_lecture_select"
-            android:text="2"
-            android:textColor="#ffffffff"
-            tools:ignore="HardcodedText" />
-
-    </LinearLayout>
-
-    <LinearLayout
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:orientation="vertical">
-
-        <Button
-            android:id="@+id/lecture_3"
-            android:layout_width="@dimen/next_lecture_select_width"
-            android:layout_height="@dimen/next_lecture_select_width"
-            android:layout_margin="1dp"
-            android:background="@drawable/next_lecture_select"
-            android:text="3"
-            android:textColor="#ffffffff"
-            tools:ignore="HardcodedText" />
-
-        <Button
-            android:id="@+id/lecture_4"
-            android:layout_width="@dimen/next_lecture_select_width"
-            android:layout_height="@dimen/next_lecture_select_width"
-            android:layout_margin="1dp"
-            android:background="@drawable/next_lecture_select"
-            android:text="4"
-            android:textColor="#ffffffff"
-            tools:ignore="HardcodedText" />
-
-    </LinearLayout>
+    <TextView
```

Comment:
I don't see any usages for this layout, is it still needed?

Reply:
Is removed now.